design-proposals/vpro-device.md

```diff
@@ -0,0 +1,196 @@
+```
+
+**Note1** - MPS requires the creation of a device before accepting CIRA connections which is part of the 2-way auth
+implemented between MPS and AMT;
+
+**Note 2** - Device Management Resource Manager will provide a `staticPassword` profile where the AMT and MEBx
+passwords are set to a well know value. Disabling this option the RM will randomly generate a password for each device.
+
+**Note 3** - Passwords are stored in `Vault` and can be always retrieved either using the Vault APIs or through the
+web-ui.
+
+### MVP Requirements
+
+At the time of writing it is expected to support the following User flows:
+
+- User is able to verify if vPRO/ISM is supported or no;
+- User is able to configure the activation of Edge Node for vPro;
+- User is able to recover the device if something goes wrong during the provisioning of the final OS;
+
+It must be carefully considered the impact on the KPIs as the User will experience worse performance when asking the
+activation of Device Management feature.
+
+However, such flow is considered not mandatory and this penalty might be accepted by the user to have in exchange extra
+manageability features.
+
+## Rationale
+
+Alternative solutions have been carefully considered and will be taken into account in future releases, in particular:
+
+- **Late Binding** where RPC is integrated in the image of the final OS. If this on one side, addresses the need to
+support remote deactivation and day2 reconfiguration, it does not provide a solution to recover the device if something
+goes wrong during the provisioning flow and require the integration with a **Platform Manageability Agent**.
+- **Cloud-Init** provisioning where AMT is activated at the end of the provisioning flow. This solution does not offer
+better integrations compared to the others and has the inherent disadvantage of not supporting feedback to the
+provisioning system that becomes completely blind on the result of the activation
+
+Both solutions provide a seamless integration of the EMT-Standalone devices. However, the first solution is considered
+more flexible and provide better control compared to the second solution.
+
+## Affected components and Teams
+
+We report hereafter the affected components and Teams:
+
+- Onboarding Manager and Tinker Actions (EIM team)
+- EMT and EMT-Tinker (EMT team)
+
+## Implementation plan
+
+Hereafter we present as steps the proposed plan to manage the device activation in the release 3.1. EIM team will
+implement the following functionality to support this design proposal:
+
+- Device discovery reports device eligibility
+- Activation will be always performed without user consent
+- EMF EN will be activated during the provisionig flow through a custom Tinker action
+- Ignition of the Platform Manageability Agent and definition of the SB gRPC APIs of the DM Resource Manager
+
+As dependency we expect the EMT team to deliver:
+
+- EMT-Tinker image supporting RPC and its dependencies
```

Comment:
specifically RCP-go ? maybe best to make the list of required elements explicit.